libs/api/repository/src/lib/gn4/auth/auth.service.ts

```diff
@@ -5,11 +5,15 @@ import { TranslateService } from '@ngx-translate/core'
 export const DEFAULT_GN4_LOGIN_URL = `/geonetwork/srv/\${lang3}/catalog.signin?redirect=\${current_url}`
 export const LOGIN_URL = new InjectionToken<string>('loginUrl')
 
+export const DEFAULT_GN4_LOGOUT_URL = `/geonetwork/signout`
+export const LOGOUT_URL = new InjectionToken<string>('loginUrl')
```

Comment:
It should be `logoutUrl` instead of `loginUrl`

Reply:
Thanks for catching this!